packages/webview_flutter/webview_flutter_android/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## NEXT
+## 2.8.4
 
 * Fixes bug preventing `mockito` code generation for tests.
+* Fixes regression where local storage wasn't cleared when `WebViewController.clearCache` was
+  called.
```

Comment:
Could you add a new integration test for this case? It seems like a very good candidate for integration testing since the high-level behavior is easily expressible in a user-visible way, so should be easy to do an integration test for.

It would also be good to add to the app-facing package, since the original mistake here demonstrates that an implementor could easily miss part of the intended behavior here.

Reply:
Done. I added it to the app-facing package as well, but it is skipped for Android. I also added a TODO to remove the skip once a new version is published.